x-pack/plugins/security_solution/public/common/utils/validators/index.test.ts

```diff
@@ -13,6 +13,10 @@ describe('helpers', () => {
       expect(isUrlInvalid('this is not a url')).toBeTruthy();
     });
 
+    test('verifies as invalid url without http(s):// prefix', () => {
+      expect(isUrlInvalid('www.thisIsNotValid.com')).toBeTruthy();
+    });
+
     test('verifies valid url', () => {
       expect(isUrlInvalid('https://www.elastic.co/')).toBeFalsy();
```

Comment:
I know you didn't write these tests but for all URL examples where possible we should use `http://www.example.com` as `example.com` is specific for tests per this RFC:
https://datatracker.ietf.org/doc/html/rfc2606#page-2